tox.ini

```diff
@@ -6,9 +6,10 @@ whitelist_externals = sphinx-build
 envlist =
     checkqa,
     readme-py27,
-    py{27,34,35,36,py}-dj{18,110,111}
-    py{34,35,36,37}-dj20
-    py{35,36,37}-dj{21,master}
+    py{27,34,35,36,py}-dj111
+    py{34,35,36,37,py3}-dj20
+    py{35,36,37,py3}-dj{21,22}
+    py{36,37,38a,py3}-djmaster
```

Comment:
py38 instead of py38a

Reply:
@blueyed I got some errors using `py38` instead of `py38a` locally, but I included that change here. If it fails CI, I'll revert to `py38a`